Scripts/NeedDisabledSVE.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+import os
+import subprocess
+import re
+import sys
+import tempfile
+import platform
+
+def ListContainsRequired(Features, RequiredFeatures):
+    return RequiredFeatures in Features
+
+def GetCPUFeatures():
+    File = open("/proc/cpuinfo", "r")
+    Lines = File.readlines()
+    File.close()
+
+    for Line in Lines:
+        if "Features" in Line:
+            Features = Line.split(":")[1].strip().split(" ")
+            return Features
+
+
+SnapdragonIDsThatSupportSVE = {
```

Comment:
Why does the "Need *disabled* SVC" script return true for CPUs that "support SVE"?